packages/language-server/src/ui5-model.ts

```diff
@@ -7,28 +7,44 @@ import {
   Json,
   TypeNameFix
 } from "@ui5-language-assistant/semantic-model";
+import { Fetcher } from "api";
 
-// TODO: use 1.71.x
-const DEFAULT_UI5_VERSION = "1.74.0";
+const DEFAULT_UI5_VERSION = "1.71.14";
+
+let fetcher: Fetcher = fetch;
+
+// These functions are for testing purposes
+/* istanbul ignore next */
+export function getFetch(): Fetcher {
```

Comment:
- What is the advantage of file(module) level state instead of optional parameter on  `getSemanticModel` with default value?

- In what scenario would you like to use these custom fetchers in the tests?

Reply:
1. I don't want it to be used outside the test for this file so I'd rather not add a parameter to a function used somewhere else. I could however add a function that receives this parameter and will not be used outside this file and its test.
1. I intended to use it in the test to fetch the files from the local tests directory instead of off the internet. The solution I wanted to do doesn't exactly work with the changes I'll do in the PR that adds tests for 1.71.14 but it could still be useful - I'll write the files to the local ignored directory so it will only happen once for each version for all tests. It will be part of the other PR anyway.

Reply:
o.k I'll look into the dev flows and downloading of resources again after its all done.

BTw: you may want to pre-download before the tetsts start running to avoid timeouts.